conversiontracerbullet/src/test/resources/server.conf

```diff
@@ -1,4 +1,4 @@
-SERVER=libdb.stanford.edu
+SERVER=db.example.com
```

Comment:
Notice how this change to the resource file entailed no consequent changes in the test code, because the test code gets access to this resource file.  Yes, it gets it via the class under test.  Maybe there's a better way to do that?  (My java-test practices are very rusty, but this PR is a good try.)